app.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+const all_animals = [];
+
+function Animal (horn){
+   this.image_url = horn.image_url;
+   this.title = horn.title;
+   this.description = horn.description;
+   this.keyword = horn.keyword;
+   this.horns = horn.horns;
+ all_animals.push(this);
+}
+
+Animal.prototype.render = function(){
+ const photo_section_html = $('#photo-template').html()
+
+ $('main').append('<section id="clone"></section>');

```

Comment:
I like that you decided to use a section tag so you have a more semantic code block (compared to using a generic div tag). Good job, keep it as a section.